ray-operator/config/samples/ray-cluster.heterogeneous.yaml

```diff
@@ -40,22 +40,20 @@ spec:
         annotations:
           key: value
       spec:
-        initContainers: # to avoid worker crashing before head service is created
+#        initContainers: # to avoid worker crashing before head service is created
```

Comment:
![ray-cluster-hetero-error](https://user-images.githubusercontent.com/37469330/135425181-36d64875-354c-4f91-99ee-fc4e5dc277e8.png)

When deploying this file in raw, above error was occured. So I deleted this field

Reply:
nice catch